README.md

```diff
@@ -1 +1,109 @@
-# ACM x DSC Server Hosting Workshop
+Welcome to the DSC and ACM intro to website hosting event!
+
+This document is meant to be worked/read through during one of our workshops, but we made sure it's possible to learn in your comfortable and socially removed home.
+
+# Topics
+
+1. Making your website.
+2. Creating your server.
+3. Installing NGINX on your server.
+4. Uploading your html to your server.
+   1. (optional) Connecting via SSH and SFTP
+5. Add to DNS.
+
+
+## Making your website.
+
+`Insert html part here`
+
+
```

Comment:
I trust that this part is about `Working with Git` where the participants clone/fork the repo.